node/src/rpc.rs

```diff
@@ -43,6 +47,36 @@ pub struct FullDeps<C, P> {
 pub fn create_full_pendulum<C, P>(
 	deps: FullDeps<C, P>,
 ) -> Result<RpcExtension, Box<dyn std::error::Error + Send + Sync>>
+where
+	C: ProvideRuntimeApi<Block>
+		+ HeaderBackend<Block>
+		+ AuxStore
+		+ HeaderMetadata<Block, Error = BlockChainError>
+		+ Send
+		+ Sync
+		+ 'static,
+	C::Api: pallet_transaction_payment_rpc::TransactionPaymentRuntimeApi<Block, Balance>,
+	C::Api: substrate_frame_rpc_system::AccountNonceApi<Block, AccountId, Nonce>,
+	C::Api: BlockBuilder<Block>,
+	C::Api: ZenlinkProtocolRuntimeApi<Block, AccountId, AssetId>,
+	P: TransactionPool + Sync + Send + 'static,
+{
+	use pallet_transaction_payment_rpc::{TransactionPayment, TransactionPaymentApiServer};
+	use substrate_frame_rpc_system::{System, SystemApiServer};
+
+	let mut module = RpcExtension::new(());
+	let FullDeps { client, pool, deny_unsafe } = deps;
+
+	module.merge(System::new(client.clone(), pool, deny_unsafe).into_rpc())?;
+	module.merge(TransactionPayment::new(client.clone()).into_rpc())?;
+	module.merge(ZenlinkProtocol::new(client).into_rpc())?;
+	Ok(module)
+}
+
+/// Instantiate all RPC extensions.
+pub fn create_full_development<C, P>(
+	deps: FullDeps<C, P>,
+) -> Result<RpcExtension, Box<dyn std::error::Error + Send + Sync>>
```

Comment:
Why do we need this now?

Reply:
Because development runtime does not implement API from zenlink.
It means that trait bound on old function does not allow because of
`C::Api: ZenlinkProtocolRuntimeApi<Block, AccountId, AssetId>,`

the solution is to duplicate the function and add trait bound for pendulum/amplitude/foucoco runtime and leave the the prev function for development runtime without changes.